### DISCUSSION TITLE
binary have no cov

### DISCUSSION THREAD

**Discussion by Jzhenli** [ChristianTremblay/BAC0#492] [Q&A]
Hi, I find analog_value have property covIncrement, when the analog_value changed, we can get the new value easily,  but if I want to subscribe binary_value, how to do it?

**Comment by ChristianTremblay** [comment]
covIncrement are required by numbers as you need to set a threshold before sending the value as "changed". I don't think binary value need this.. it is true or false... so as soon as the value change, the condition is met.
